locale/fi/LC_MESSAGES/django.po

```diff
@@ -1323,81 +1323,89 @@ msgstr "Muokkaa moduulia"
 msgid "Open module"
 msgstr "Avaa moduuli"
 
-#: edit_course/templates/edit_course/edit_content.html:109
+#: edit_course/templates/edit_course/edit_content.html:111
 #: exercise/templates/exercise/exercise_base.html:91
 msgid "Edit exercise"
 msgstr "Muokkaa tehtävää"
 
-#: edit_course/templates/edit_course/edit_content.html:115
+#: edit_course/templates/edit_course/edit_content.html:111
+msgid "Edit content chapter"
+msgstr "Muokkaa kappaletta"
```

Comment:
We have been uncertain about how to translate "content chapter" to Finnish. Sometimes it is called "sisältökappale". "Kappale" alone does not feel sufficient. You should use "sisältökappale" here unless we get better ideas.